tests/integration/cli.py

```diff
@@ -15,21 +15,24 @@ def test_add_list_delete_projects():
 
     The project list does not need to be empty before running the test."""
     output = subprocess.check_output(['haas', 'list_projects'])
-    projects = json.loads(output)
+    projects = output.split(" ")
     assert PROJECT1 not in projects
     assert PROJECT2 not in projects
+    projects.remove('\n')  # remove \n from list
     size = len(projects)
 
     subprocess.check_call(['haas', 'project_create', PROJECT1])
     output = subprocess.check_output(['haas', 'list_projects'])
-    projects = json.loads(output)
+    projects = output.split(" ")
+    projects = map(str.strip, projects)  # strips off \n from project name
```

Comment:
also, we could do  `projects.remove('\n')` since we want to remove `\n` here. I don't remember why I used map.